src/System.Memory/tests/Performance/Perf.Span.IndexOf.cs

```diff
@@ -209,5 +209,19 @@ public void SpanIndexOfSpanComparison(string input, string value, StringComparis
                     inputSpan.IndexOf(valueSpan, comparisonType);
                 }
         }
+
+        [Benchmark]
+        [MemberData(nameof(s_indexTestData))]
+        public void SpanLastIndexOfSpanComparison(string input, string value, StringComparison comparisonType)
+        {
+            ReadOnlySpan<char> inputSpan = input.AsSpan();
+            ReadOnlySpan<char> valueSpan = value.AsSpan();
+
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                {
+                    inputSpan.LastIndexOf(valueSpan, comparisonType);
```

Comment:
Do you think it is useful to also add the string-based perf tests as baseline to compare against?

Reply:
I would prefer not duplicating existing benchmarks. We definitely have some in System.Runtime?

Reply:
I see. In that case, I agree, we shouldn't duplicate them. During the port to BDN though, I think we should keep the string/span perf tests together so we can compare the results easily.